test/NuGet.Core.Tests/NuGet.Versioning.Test/NuGetVersionTest.cs

```diff
@@ -363,5 +417,36 @@ public void TryParseReturnsFalseWhenUnableToParseString(string versionString)
             Assert.False(result);
```

Comment:
If my understanding is correct, the current implementation returns false when version number is "19". However, the with the current implementation it looks like it returns true. please verify.